src/lib/components/LivestreamContainer/index.js

```diff
@@ -15,11 +15,12 @@ class LivestreamContainer extends BaseStateElement {
     super();
     this.videoId = null;
     this.isChatActive = true;
+    this._isSignedIn = undefined;
```

Comment:
```suggestion
    this.isSignedIn = undefined;
```

I guess to be consistent with the other properties.